indexers/src/services/relayer.service.ts

```diff
@@ -89,38 +83,42 @@ export class RelayerService {
   }
 
   /**
-   * Computes relayer specific metrics from the handlePostRequest transaction
+   * Computes relayer specific stats from the PostRequestHandled/PostResponseHandle event's transaction
    */
-  static async handlePostRequestTransaction(
-    transaction: HandlePostRequestsTransaction,
+  static async handlePostRequestOrPostResponseHandledEvent(
```

Comment:
Where do we update the request status when processing `PostRequestHandled` and `PostResponseHandled` events?

Reply:
This has been added in the handler for `PostRequestHandled` and `PostResponseHandled`